adr/cli.py

```diff
@@ -100,7 +101,9 @@ def run_recipe(recipe, args, fmt='table'):
     mod = importlib.import_module(modname, package='adr')
     output = mod.run(args)
 
-    if isinstance(fmt, string_types):
+    if output == MissingDataError:
+        return 'ActiveData didn\'t return any data.'
```

Comment:
After fixing the other comment, you can put a `try/except` block around the call to `mod.run`. You can catch the custom exception you created and then print the error message in the `except` block. Hope that makes sense, and feel free to clarify!

Reply:
Sounds good! Raising an appropriate exception makes sense. I had to step away for a few days, so now there are conflicts between my PR and other updates in the meantime. Should I re-fork and resubmit or is it an easy resolution? I've not run into merge conflicts before, but the changes were simple enough that I can totally repeat if necessary.

Reply:
No problem for stepping away, thanks for sticking with it! Let me try and resolve the conflicts for you, then you can continue working here.